packaging/_elffile.py

```diff
@@ -0,0 +1,99 @@
+"""ELF file parser.
```

Comment:
The docstring style in the repo is a blank line after the opening triple quote.

```suggestion
"""
ELF file parser.
```